modules/openy_features/openy_media/src/Plugin/Field/FieldFormatter/FileIframeFormatter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\openy_media\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Url;
+use Drupal\file\Plugin\Field\FieldFormatter\FileFormatterBase;
+
+/**
+ * Plugin implementation of the file iframe field formatter.
+ *
+ * @FieldFormatter(
+ *   id = "openy_file_iframe",
+ *   label = @Translation("File Iframe"),
+ *   field_types = {
+ *     "file"
+ *   }
+ * )
+ */
+class FileIframeFormatter extends FileFormatterBase {
```

Comment:
This class should live in `openy_media_document`, because it's being used only there.